src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -443,4 +443,28 @@ private static void updateTrackingStartTime(long updateTime)
 
     isFile1CurrTimeTrackerFile = !isFile1CurrTimeTrackerFile;
   }
+
+  /**
+   * Advance Historical FIFO tracking time. The intention is for exception handlers to make the
+   * decision to advance time.
+   *
+   * @throws IOException if unable to read or write files
+   * @throws TimeTrackerUnrecoverableException if both time tracking files are corrupted
+   * @throws CorruptedTimeTrackerException one of the tracking files is corrupted
+   */
+  public static void advanceTrackingStartTime()
+      throws CorruptedTimeTrackerException, IOException, TimeTrackerUnrecoverableException {
+
+    // Get start time from file, or start new time tracker if startNewTimeTracker is true.
```

Comment:
This comment needs to be updated. startNewTimeTracker doesn't exist here

Reply:
changed.